ts/Series/XRange/XRangeSeries.ts

```diff
@@ -259,6 +259,14 @@ class XRangeSeries extends ColumnSeries {
     public alignDataLabel(point: XRangePoint): void {
         const oldPlotX = point.plotX;
         point.plotX = pick(point.dlBox && point.dlBox.centerX, point.plotX);
+
+        if (point.dataLabel && point.shapeArgs && point.shapeArgs.width) {
+            const width = point.shapeArgs.width;
+            point.dataLabel.css({
+                width: `${width}px`
```

Comment:
This can be a little bit more compact if we skip the `const` and just enter `point.shapeArgs.width` directly.